src/main/g8/README.md

```diff
@@ -14,15 +13,27 @@ The intention is to use `sbt new` on this library's exterior g8 template, then t
 - `sbt-native-packager` plugin for **building a linked binary** _(builds ahead-of-time using GraalVM statically linking and building all dependencies in a docker image via `sbt graalvm-native-image:packageBin`)_
 - `scalatest` for **testing** _(default test runner via `sbt test`)_
 - `sbt-scoverage` for **test coverage** _(statement-level coverage via `sbt coverage`)_
-- `scalalogging -> slf4j -> log4j2` for **logging** _(scala code -> interface -> mechanism)_
+- `scalalogging -> logback-classic` for **logging** _(`logback-classic` is used as a backend)_
 - `scopt` for command-line **argument parsing**
 - `sbt-updates` for **keeping up-to-date** _(dependency updates via `sbt dependencyUpdates`)_
 - `sbt-dependency-graph` for **understanding your package** _(dependency graph via `sbt dependencyBrowseGraph` and dependency stats via `sbt dependencyStats`)_
 
-Check out the [Libs](#markdown-header-libs) below for more details on each.
+Check out the [Libs](#libs) below for more details on each.
 
 # How to... #
 
+## Prerequisites
+
+There are a couple of things you're probably gonna want to have installed on your machine before you go any further.
+
+- SBT: Used for building, testing and inspecting the project.
+    - Install on [OSX](https://www.scala-sbt.org/1.x/docs/Installing-sbt-on-Mac.html)
+    - Install on [Windows](https://www.scala-sbt.org/1.x/docs/Installing-sbt-on-Windows.html)
+    - Install on [Linux](https://www.scala-sbt.org/1.x/docs/Installing-sbt-on-Linux.html)
+- Docker: Wraps the entire build process to make your life easier.
+    - Install via [Docker Desktop](https://hub.docker.com/?overlay=onboarding)
+    - If you only intend to run this project on your own machine (which you can do by [removing this line](https://github.com/sbchapin/quickstart-scala-sbt-native-image.g8/blob/master/src/main/g8/build.sbt#L20)) you won't need Docker so feel free to skip this. 
```

Comment:
This latter bit is env-specific.  Technically, you don't need to remove it if you have libc, which would be most linux environments.

Suggested edit:
```markdown
- Docker: Wraps the entire build process to make your life easier. This means you don't need GraalVM or native-image in your environment.
    - Install via [Docker Desktop](https://hub.docker.com/?overlay=onboarding)
- No Docker: If you intend to build this project to a binary on your machine rather than build it within Docker...
    - Install [GraalVM CE](https://www.graalvm.org/docs/getting-started/) - or better yet - [install jabba](https://github.com/shyiko/jabba#installation) Java Version Manager to [install GraalVM](https://github.com/shyiko/jabba#usage)
    - Install [native-image](https://www.graalvm.org/docs/reference-manual/native-image/) via `gu install native-image`
    - [Remove this line](https://github.com/sbchapin/quickstart-scala-sbt-native-image.g8/blob/master/src/main/g8/build.sbt#L20) if you are not running on linux _(and no, OSX is not linux)_
```

Reply:
I like this a lot 👍 with the exception of all the negatives on this line:
`if you are *not* running on linux _(and *no*, OSX is *not* linux)_`

I switched it based on your comment above. Hopefully it looks okay to you?

Reply:
❎